README.md

```diff
@@ -46,6 +46,12 @@ ansible-playbook opentermsarchive.deployment.<playbook-name> --check --diff
 
 > See “[Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html)” in Ansible’s user guide for more information about Ansible collections.
 
+### Versions compatibility
+
+| Deployment | Engine | Federated API |
+| --- | --- | --- |
+| 0.1.0 | >= 0.36.0 | >= 0.1.0 |
```

Comment:
I am concerned that this is very likely to become outdated. I would feel more comfortable if the dependency was made clear in the dependent (i.e. engine and Federated API) rather than in here.

Reply:
After IRL discussion, we will remove it for now to avoid outdated documentation